subjects/java/checkpoints/is-anagram/README.md

```diff
@@ -0,0 +1,45 @@
+## Is Anagram
+
+### Instructions
+
+Create a class `AnagramChecker` that provides a method to check if two strings are anagrams. An anagram is a word or phrase formed by rearranging the letters of a different word or phrase, typically using all the original letters exactly once.
+
+### Expected Class
+
+```java
+public class AnagramChecker {
+    public boolean isAnagram(String str1, String str2) {
+        // Implementation to check if str1 and str2 are anagrams
+    }
+}
+```
+
+### Usage
+
+Here is a possible ExerciseRunner.java to test your class:
```

Comment:
A minor comment: it would be nice to have `ExerciseRunner.java` formatted as the other subjects

Reply:
done